package.json

```diff
@@ -5,10 +5,13 @@
   "main": "index.js",
   "scripts": {
     "test": "jest --forceExit",
-    "seed": "node db/seedData.js",
+    "seed": "node db/mongo/seedData.js",
     "start": "nodemon server/index.js",
     "create": "node db/modelsPostgres/Game.js",
-    "dataGen": "node --max-old-space-size=16384 db/dataGeneration.js"
+    "dataGenGame": "node --max-old-space-size=16384 db/dataGen/dataGameGeneration.js",
```

Comment:
Might have to careful using this considering they will be running on EC2 instances with limited RAM